appdata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.geogebra.GeoGebra.desktop</id>
+  <name>GeoGebra</name>
+  <developer_name>GeoGebra</developer_name>
+  <summary>Dynamic mathematics software</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
Yeah as I added in the commit message: GeoGebra license is non-commercial. See https://www.geogebra.org/license for details.

In the freedesktop.org site I couldn't find an option for `<project_license>` that matches. The site refers to https://spdx.org/licenses/ but those are all open licenses.